EpikCord/__init__.py

```diff
@@ -823,8 +823,10 @@ async def handle_interaction(self, interaction):
 
                 for option in interaction.options:
                     options.append(option.get("value"))
-
-            return await command.callback(interaction, *options)
+            try:
+                return await command.callback(interaction, *options)
+            except Exception as e:
+                await self.command_error(e, interaction=interaction)
```

Comment:
I think something that could be nice is making a `CommandError` and/or a `InteractionError` having some command/interaction attribute, instead of passing both.

That way you could do the following:
```py
async def command_error(self, error:Exception):
    if isisnstance(error, CommandError):
        print(error.command) 
```
And mypy would be happy about this too :D

Reply:
nah, we let the user define